### PR TITLE
auto OCP-76051

### PR DESCRIPTION
**Pull request by yapei**
Add one small upgrade scenario. Since cypress tests are all skipped on vsphere profile the new TC can only be automated in cucushift

### CODE REVIEW

**Comment by yapei** [comment]
post check pass log Runner/1040794/console

File: features/upgrade/web/console-upgrade.feature [hunk 1]
```diff
@@ -88,3 +88,25 @@ Feature: web console related upgrade check
...
       | resource_name | hello-daemonset |
     Then the step should succeed
+
+  # @author yapei@redhat.com
+  # @case_id OCP-76051
+  @upgrade-check
+  @admin
+  @console
+  @users=upuser1,upuser2
+  @4.16 @4.15 @4.14 @4.13 @4.12 @4.11 @4.10 @4.9 @4.8 @4.7 @4.6
```
Comment (liangxia) [inline features/upgrade/web/console-upgrade.feature:98]: The rules are added for 4.14 to 4.18, while tags here does not have 4.17 and 4.18.  Is it expected ?

Comment (yapei) [reply features/upgrade/web/console-upgrade.feature:98]: good catch, let me update

**Pull request merged**